docs/content-services/components/alfresco-viewer.component.md

```diff
@@ -67,7 +67,8 @@ See the [Custom layout](#custom-layout) section for full details of all availabl
 | ---- | ---- | ------------- | ----------- |
 | allowDownload | `boolean` | true | Toggles downloading. |
 | allowFullScreen | `boolean` | true | Toggles the 'Full Screen' feature. |
-| allowGoBack | `boolean` | true | Allows `back` navigation |
+| allowGoBack | `boolean` | true | Allows `back` navigation. |
+| closeButtonPosition | `string` | `left` | Set close button position right/left. |
```

Comment:
instead of string should be CloseButtonPosition type

Reply:
Done